src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/InputInProcess.cs

```diff
@@ -4,21 +4,29 @@
 
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.VisualStudio.Extensibility.Testing;
 using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
 using Microsoft.VisualStudio.Threading;
+using Roslyn.Utilities;
 
 namespace Roslyn.VisualStudio.IntegrationTests.InProcess
 {
-    internal class InputInProcess : InProcComponent
+    [TestService]
+    internal partial class InputInProcess
     {
-        public InputInProcess(TestServices testServices)
-            : base(testServices)
+        private SendKeysImpl? _lazySendKeys;
+
+        private SendKeysImpl SendKeys
         {
-            SendKeys = new(testServices);
+            get
+            {
+                return LazyInitialization.EnsureInitialized(
+                    ref _lazySendKeys,
+                    static testServices => new SendKeysImpl(testServices),
+                    TestServices);
```

Comment:
📝 This will move to `InitializeCoreAsync` after microsoft/vs-extension-testing#96 is merged

Reply:
➡️ Now fixed